includes/definitions/cyberpower.yaml

```diff
@@ -0,0 +1,7 @@
+os: cyberpower
+text: 'Cyberpower'
+type: power
+over:
+    - { graph: device_current, text: Current }
```

Comment:
Nope. The SNMP implementation on it is less than complete. It reports having a interface named agent as can be seen in the walk above, but it is non-functional(or atleast is on the PDU15SW8RNET).

Speaking as some one who does datacenter work for a living, it is a PDU, current is the big primary one to be concerned with for it.